### PR TITLE
Adds KI to release notes for IPv6 egress IPs

### PR DESCRIPTION
**Pull request by stevsmit**
Last minute known issue request

Version(s):
4.14

Issue:
None. Direct message

Link to docs preview:
https://67018--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-known-issues

No needed.

### CODE REVIEW

**Comment by michaelryanpeter** [comment]
lgtm
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -2121,7 +2121,7 @@ In the following tables, features are marked with the following statuses:
 |Egress IPs on additional network interfaces
 |Not Available
 |Not Available
-|Technology Preview
+|General Availability
```
Comment (mjpytlak) [inline release_notes/ocp-4-14-release-notes.adoc:2124]: @stevsmit and @JoeAldinger At Kathryn's request I am going to merge this. However, either in a separate PR or possibly a RN clean up PR, please remove this feature from the Tech Preview table. It is no longer necessary to reference it here, seeing as the feature is going straight to GA.

Comment (JoeAldinger) [reply release_notes/ocp-4-14-release-notes.adoc:2124]: @mjpytlak cleaned up in [this](https://github.com/openshift/openshift-docs/pull/67048) PR

**Pull request merged**